etk/extractors/dbpedia_spotlight_extractor.py

```diff
@@ -40,14 +45,49 @@ def combiner(self, results: dict) -> List[Extraction]:
             resources_results = results["Resources"]
             for one_result in resources_results:
                 types = one_result['@types'].split(',')
-                return_result.append(Extraction(confidence=float(results['@confidence']),
-                                                extractor_name=self.name,
-                                                start_char=int(one_result['@offset']),
-                                                end_char=int(one_result['@offset']) + len(
-                                                    one_result['@surfaceForm']),
-                                                value={'surface_form': one_result['@surfaceForm'],
-                                                       'uri': one_result['@URI'],
-                                                       'types': types,
-                                                       'similarity_scores': float(one_result['@similarityScore'])}))
+                if self.get_attr:
+                    attr = self.attr_finder(one_result['@URI'])
+                    return_result.append(Extraction(confidence=float(results['@confidence']),
```

Comment:
This and line #61 : code is repetitive, based on get_attr, only add `'attributes': attr`